### ISSUE TITLE
ui5-wizard: JAWS is not reading the content by using the down arrow

### ISSUE THREAD

**Issue by GongRichard** [SAP/ui5-webcomponents#3153]
**Describe the bug**
JAWS is not reading the content by using the down arrow for ui5-wizard step tab.

**To reproduce**

1. Use Chrome open page https://github.wdf.sap.corp/pages/xweb/goalmanagementx/app.html?mock#/goal-list
2. click the 'Skip Tour' button
3. click the Cascade Goal button on the top right of the header
4. click to focus step 1 "Select Goals"
5. Validate the page using the Down arrow key and JAWS is not reading the non-interactive element using the down arrow

**Expected behavior**
The wizard step should read by the screen reader, when the user will press the arrow key immediately the screen reader should announce the next or previous content. 

**Context**
- UI5 Web Components version: 0.29.13
- OS/Platform: {Windows}
- Browser *(if relevant)*: {Chrome}
- Other information: {...}

Organization:
SF

Priority: (optional) (Low, Medium, High or Very High)
High

**Comment by ilhan007** [comment]
Hi @GongRichard there was accessibility improvement in the Wizard that has been released with 0.29.14. 
I saw that you refer to 0.29.13
Could you test with the latest version to avoid any unnecessary efforts, might be different readings.

**Comment by GongRichard** [comment]
Hi @ilhan007,

OK, I will test it again after we upgrade to 0.29.14.

Thanks,
Richard

**Comment by GongRichard** [comment]
Hi @ilhan007,

OK, I have check this issue in 0.29.14, it still exists. Please have a check and see whether it can be fixed in 0.29.15.

Thanks,
Richard

**Comment by alexandar-mitsev** [comment]
Hello @SAP/ui5-webcomponents-topic-rl ,

Can you take a look at the issue. I think that the requested behaviour is not according to the ACC specification, but please check and confirm.

Regards,
Alexandar Mitsev

**Comment by elenastoyanovaa** [comment]
Hello @GongRichard ,

Non-interactive elements are read out with the reading mode. In order to switch on the reading mode you should use Insert+Z and hear the following speech output: "Virtual Cursor On". Then, you could navigate with the use of the following keystrokes: https://doccenter.freedomscientific.com/doccenter/doccenter/rs11f929e9c511/2012-04-24_teachersandtrainers-l7/02_jawsandmagicreadingcommandsandcursors.htm
If you still reproduce the issue after following those steps, please return the issue with the following information:
- A screen recording, showing the steps to reproduce and that the issue is reproduced
- Exact JAWS version. I would recommend you to use the latest JAWS 2021. In this way you could ensure that some JAWS issues could be fixed

Kind Regards,
Elena

**Comment by GongRichard** [comment]
Hi @elenastoyanovaa,

I think it's a ui5-wizard issue. 
If you tab to focus "Select All" checkbox, then JAWS is reading the non-interactive element using the down arrow.
![2021-04-23_13-26-25](https://user-images.githubusercontent.com/13098801/115822954-01c8e900-a438-11eb-9de2-ac26fe03cd68.jpg)
But if you tab to focus step 1 "Select Goals", then JAWS is not reading the non-interactive element using the down arrow.
![2021-04-23_13-27-08](https://user-images.githubusercontent.com/13098801/115823054-39d02c00-a438-11eb-8c04-03e040212f05.jpg)

BTW, I am using the latest JAWS 2021. 

Thanks,
Richard

**Comment by elenastoyanovaa** [comment]
Hi @GongRichard ,

You focus on this checkbox and JAWS somehow automatically switches modes - it switches from application to reading (virtual cursor) mode, which is rather not expected. The wizard component behaves as expected. 

If you read the content via JAWS and in the same time interact with the application (type, focus using TAB, activate buttons with SPACE/ENTER and so on) you use the screen reader in application mode. When in application mode, reading non-interactive elements is not possible/ not expected. So being able to navigate with arrow keys and reading out non-interactive elements after focusing to a checkbox (after interaction) is a problem with the screen reader, related to wrong mode switching, not and issue, or desired behaviour of a component.

As I mentioned in the previous comment - to read out the page or an element in the page, you should ensure you have switched to the correct mode via Ins+Z and navigate with the keys described in the link provided.

Kind Regards,
Elena

**Comment by Arjuncse07** [comment]
Hello, Community I am strugguling with a problem for the Web Accessiblity.  We Have a SSN field in which the user will enter the SSN Number, if SSN Number is wrong, it will shows an Error message please Enter three digit number dash two digit number dash four digit number XXX-XX-XXXX.    When I traverse the webpage threw TAB key, NVDA will readout the error message when user enters wrong SSN , but when i use JAWS it does't read error message. Please Guide
